packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationModule.kt

```diff
@@ -39,6 +42,21 @@ class LocalizationModule : Module() {
     Function("getCalendars") {
       return@Function getCalendars()
     }
+
+    OnCreate {
+      if (appContext.reactContext != null) setRTLFromStringResources(appContext.reactContext!!)
+    }
+  }
+
+  private fun setRTLFromStringResources(context: Context) {
+    // These keys are used by React Native here: https://github.com/facebook/react-native/blob/main/React/Modules/RCTI18nUtil.m
+    // We set them before React loads to ensure it gets rendered correctly the first time the app is opened.
+    val supportsRTL = appContext.reactContext?.getString(R.string.ExpoLocalization_supportsRTL)
+    if (supportsRTL != "true" && supportsRTL != "false") return
```

Comment:
```suggestion
    val supportsRTL = appContext.reactContext?.getString(R.string.ExpoLocalization_supportsRTL) ?: return
```

Reply:
It can also be unset when it's using the default strings.xml value

Reply:
Ohh, I see. I thought it will be null in that case.